src/components/ComposerAttachments.vue

```diff
@@ -98,7 +98,7 @@ export default {
 				fileName: name,
 				displayName: trimStart('/')(name),
 				id,
-				isLocal: id !== undefined,
+				type: id !== undefined ? 'local' : 'cloud',
```

Comment:
this change will conflict with https://github.com/nextcloud/mail/pull/3998. The PR that is merged later has to be adjusted

Reply:
done